README.md

```diff
@@ -99,3 +100,5 @@ To build and run the project using Docker for local development:
 Replace `<container_id>` with the actual container ID, which you can get by running `docker ps`.
 
 Note: For hot-reloading in development, you may need to adjust the Dockerfile and use a tool like nodemon.
+=======
+>>>>>>> Stashed changes
```

Comment:
Same here.

Reply:
Oops, I removed it, I guess my editor play error on me.